src/main/resources/view/LoanListCard.fxml

```diff
@@ -7,30 +7,84 @@
+      <HBox alignment="CENTER_LEFT" spacing="5">
+            <TextFlow>
+               <children>
+                  <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="Loan ID: ">
+                     <font>
+                        <Font size="14.0" />
+                     </font></Text>
+              <Label fx:id="id" styleClass="cell_big_label">
+                <minWidth>
+                  <!-- Ensures that the label text is never truncated -->
+                  <Region fx:constant="USE_PREF_SIZE" />
+                </minWidth>
+              </Label>
+               </children>
+            </TextFlow>
       </HBox>
       <FlowPane fx:id="tags" />
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-      <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+         <TextFlow>
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="Bike: " />
+            <Label fx:id="bike" styleClass="cell_small_label" text="\$bike" />
+            </children>
+         </TextFlow>
+         <TextFlow>
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="Loaned to: " />
+           <Label fx:id="name" styleClass="cell_small_label" text="\$name" />
+            </children>
+         </TextFlow>
+         <TextFlow prefHeight="0.0" prefWidth="130.0">
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="  &gt; Nric: " wrappingWidth="120.21043395996094" />
+               <Label fx:id="nric" styleClass="cell_small_label" text="\$nric" />
+            </children>
+         </TextFlow>
+         <TextFlow>
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="  &gt; Phone: " />
+               <Label fx:id="phone" text="\$phone" />
+            </children>
+         </TextFlow>
+         <TextFlow>
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="  &gt; Email: " />
+               <Label fx:id="email" text="\$email" />
+            </children>
+         </TextFlow>
+         <TextFlow>
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="On " wrappingWidth="27.21875" />
+            <Label fx:id="time" styleClass="cell_small_label" text="\$time" />
+            </children>
+         </TextFlow>
+         <TextFlow prefHeight="17.0" prefWidth="130.0">
+            <children>
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="With rate: " />
+            <Label fx:id="rate" prefHeight="17.0" prefWidth="33.0" styleClass="cell_small_label" text="\$rate" />
+               <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="/hr" />
```

Comment:
Hard coding this seems really bad. If the rate is 1.2345678, you might get text over your `/hr` text.

Reply:
If you are unable to resolve this, can we set rate to only accept 2 decimal place?

Reply:
The rate is alr limited to 2 decimal places.

Reply:
Roger